docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_WideAndDeep_Family_Models.md

```diff
@@ -1,16 +1,16 @@
-# Convert TensorFlow Wide and Deep Family Models {#openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_WideAndDeep_Family_Models}
+# Converting TensorFlow Wide and Deep Family Models {#openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_WideAndDeep_Family_Models}
 
 The Wide and Deep models is a combination of wide and deep parts for memorization and generalization of object features respectively.
 These models can contain different types of object features such as numerical, categorical, sparse and sequential features. These feature types are specified
-through Tensorflow* tf.feature_column API. Table below presents what feature types are supported by the OpenVINO&trade; toolkit.
+through Tensorflow tf.feature_column API. Table below presents what feature types are supported by the OpenVINO&trade; toolkit.
 
 | numeric | (weighted) categorical | categorical with hash | bucketized | sequential | crossed |
 |:-------:|:----------------------:|:---------------------:|:----------:|:----------:|:-------:|
 | yes     | yes                    | no                    | yes        | yes        | no      |
 
-**NOTE**: the categorical with hash and crossed features are currently unsupported since The OpenVINO&trade; toolkit does not support tensors of `string` type and operations with them.
+> **NOTE**: The categorical with hash and crossed features are currently unsupported since the OpenVINO&trade; toolkit does not support tensors of *`string`* type and operations with them.
```

Comment:
```suggestion
> **NOTE**: The categorical with hash and crossed features are currently unsupported since the OpenVINO toolkit does not support tensors of *`string`* type and operations with them.
```